src/expressions.md

```diff
@@ -88,9 +88,26 @@ created and used instead. The lifetime of temporary values is typically the
 innermost enclosing statement; the tail expression of a block is considered
 part of the statement that encloses the block.
 
-When a temporary rvalue is being created that is assigned into a `let`
-declaration, however, the temporary is created with the lifetime of the
-enclosing block instead, as using the enclosing statement (the `let`
+A first exception is when a temporary value is created in the
```

Comment:
You are right: it is odd. I don't know if there are still some exceptions missing, however.

Can someone confirm that there are no other exceptions missing? Then we can add "There are two exceptions to this rule."